snscrape/modules/instagram.py

```diff
@@ -53,24 +55,28 @@ def __init__(self, **kwargs):
 		super().__init__(**kwargs)
 		self._headers = {'User-Agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/39.0.2171.95 Safari/537.36'}
 		self._initialPage = None
+		self._api_url = None
```

Comment:
I think camelCase should be used for variables?

Reply:
I believe the preferred python syntax is not camel case, but this repo does use it. [PEP-8](https://peps.python.org/pep-0008/#function-and-variable-names)